lib/cocoapods-downloader/http.rb

```diff
@@ -8,7 +8,14 @@ class Http < RemoteFile
       executable :curl
 
       def download_file(full_filename)
-        curl! '-f', '-L', '-o', full_filename, url, '--create-dirs', '--netrc-optional', '--retry', '2'
+        parameters = ['-f', '-L', '-o', full_filename, url, '--create-dirs', '--netrc-optional', '--retry', '2']
+
+        headers.each do |h|
+          parameters << '-H'
+          parameters << h
+        end unless headers.nil?
+
+        curl! parameters
```

Comment:
Could you add a test for this? Something that verifies the options passed to `curl!` would be great. There are examples in `http_spec.rb`

Reply:
@amorde Have pushed 2 tests, 1 to check the default `curl` parameters, another to check for supplied `:headers` specifically. I'm not amazing at Ruby, happy to take suggestions on the code and/or tests!